lightning/src/ln/channelmanager.rs

```diff
@@ -173,6 +173,22 @@ struct ClaimableHTLC {
 	onion_payload: OnionPayload,
 }
 
+/// A payment identifier used to correlate an MPP payment's per-path HTLC sources internally.
+#[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
+pub(crate) struct MppId(pub [u8; 32]);
```

Comment:
Maybe `MppPartId`?

Reply:
This is used to correlate multiple MPP parts, there isn't an `MppId` per-part, if that makes sense

Reply:
Ah, I see. Thanks for explaining.